### ISSUE TITLE
Security: Open redirect

### ISSUE THREAD

**Issue by da2x** [haystack/tipsy-site#5]
Open redirects are a security threat and redirect.php is wide open.

Why is it even needed? In any case, it must be closed. Limiting it to paypal.com can be a stop-gap measure to closing it. However, the redirect also isn’t used over HTTPS, so the best thing is to just get rid of it.

**Comment by karger** [comment]
I'm honestly not sure what this redirect is being used for; I'm leery of getting rid of it till I understand whether it would break anything.

**Comment by da2x** [comment]
The client redirects all links to PayPal and Dwolla through the redirect server. Presumably to collect user data? Can’t really see any other reason for it.

**Comment by karger** [comment]
Right.  My point is that it's not right to tell publishers to provide a 
bitcoin address before the extension can actually pay to it---otherwise 
publishers won't get paid.  So I'll merge this in once the change to the 
extension is deployed.
